src/lib.rs

```diff
@@ -40,6 +38,21 @@ pub const ATTEMPT_COUNTER_DEFAULT_RETRIES: u8 = 8;
 /// Do not make longer messages than this size
 pub const CTAPHID_MESSAGE_SIZE_LIMIT: usize = 3072;
 
+#[repr(u8)]
+pub enum CustomStatus {
+    /// The custom status to be shown for the successful verification for the Reverse HOTP.
+    /// By design this should be: blink green LED for 10 seconds, highest priority
+    ReverseHotpSuccess = 0,
+
+    /// The custom status to be shown for the failed verification for the Reverse HOTP.
+    /// By design this should be: blink red LED infinite times, highest priority
+    ReverseHotpError = 1,
+}
```

Comment:
These should be configuration options so that the runner can provide the status values to use.

Reply:
That would be best indeed. Otherwise we would have to share a crate with these Enums.
Do you have some initial idea how this could look like?

Reply:
I would just add it to `authenticator::Options`.  There are no sensible defaults so we would have to remove the `Default` implementation for `Options`, remove the `Authenticator::with_options` method and add `options` to `Authenticator::new` as a required argument.

https://github.com/Nitrokey/trussed-secrets-app/blob/07b4b87b364372fb13024870fd6aae0808cafd17/src/authenticator.rs#L24-L27
https://github.com/Nitrokey/trussed-secrets-app/blob/07b4b87b364372fb13024870fd6aae0808cafd17/src/authenticator.rs#L185-L195

Reply:
Sounds good. On it.